device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-C64/buffers_dynamic.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers_dynamic.json.j2
```

Comment:
if the SKU is not expected to support dynamic buffer calculation do we still need to have the file?
@stephenxs  can you please comment on that?

Reply:
@liat-grozovik 
In theory, we don't need this file given that the dynamic buffer calculation is disabled in this SKU.
But I suggest keeping it to make it consistent with other SKUs - we have this file in all SKUs regardless of whether dynamic buffer calculation is enabled.